webpack.config.js

```diff
@@ -9,6 +9,8 @@ function getParsedEnv() {
       ...env.parsed,
       VERSION: version,
       IS_EU_ITERABLE_SERVICE: process.env.IS_EU_ITERABLE_SERVICE || false,
+      DANGEROUSLY_ALLOW_JS_POPUP_EXECUTION:
+        process.env.DANGEROUSLY_ALLOW_JS_POPUP_EXECUTION || false
```

Comment:
Should we do a strict check against `true` given the effect of the changes this value controls?

Reply:
this is the webpack config to allow the env variable to be passed in. made the check a const and made it stricter [here](https://github.com/Iterable/iterable-web-sdk/pull/380/files#diff-8fa4b52909f895e8cda060d2035234e0a42ca2c7d3f8f8de1b35a056537bf199R10)